src/models/tiles/tile-model.ts

```diff
@@ -87,7 +87,8 @@ export const TileModel = types
     },
     exportJson(options?: ITileExportOptions): string | undefined {
       const { includeId, excludeTitle, ...otherOptions } = options || {};
-      let contentJson = (self.content as any).exportJson(otherOptions);
+      // let contentJson = (self.content as any).exportJson(otherOptions);
+      let contentJson = (self.content as any).exportJson ? (self.content as any).exportJson(otherOptions) : undefined;
```

Comment:
```suggestion
      let contentJson = (self.content as any).exportJson?.(otherOptions);
```

Are there some tile content models that don't have an exportJson?

Reply:
@scytacki I don't think so, though I'm not 100% sure.

All of these tile types have an exportJson:

geometry
image
table
text
datacard
dataflow
diagram-viewer
drawing
graph

These currently don't:

dataflow-tool -- I don't think this is actually a tile...or maybe it's static documentation that can be displayed as a tile?
expression -- this is a new tile type that isn't complete